### DISCUSSION TITLE
after manual install is omz update sufficient

### DISCUSSION THREAD

**Discussion by Rhahkeem** [ohmyzsh/ohmyzsh#11220] [Help] [answered]
I installed ohmyzsh in the standard ~/.oh-my-zsh directory, but looking at the git history i'm pretty out of date
From git log
```
commit 5609079fae4357b69f887e1210092c2e653ab2da
Author: thinszx <1217641779@qq.com>
Date:   Thu Mar 17 19:54:47 2022 +0800

    fix(updater): change remote using deprecated `git:` protocol (#10779)

```

is `omz update` sufficient? or should I be trying to also pull down the commits from upstream/remote ?

**Comment by mcornella** [comment, marked as answer]
`omz update` is enough. There was a typo that prevented automatic updates, if you update manually you'll get the fix and won't need to manually update anymore.

**Reply by Rhahkeem** [reply]
how long ago was the typo fixed? Should I do one round of git pulling and should be good from now on?

**Comment by Rhahkeem** [comment]
Turns out I messed up the majority of my git history somehow so `omz update` is the safest option
